pkg/util/runtime/features.go

```diff
@@ -36,16 +36,21 @@ const (
 
 	// FeatureContainerPortAllocation is a feature flag to enable/disable allocating ports to several containers in a pod
 	FeatureContainerPortAllocation Feature = "ContainerPortAllocation"
+
+	// FeatureFixRollingUpdateScaleDown is a feature flag to enable/disable rolling update fix of scale down, when ReadyReplicas
+	// count is taken into account
+	FeatureFixRollingUpdateScaleDown Feature = "FixRollingUpdateScaleDown"
```

Comment:
```suggestion
	FeatureRollingUpdateOnReady Feature = "RollingUpdateOnReady"
```

WDYT?

Also, should we add some docs about this somewhere? So it's clear the strategy?

Reply:
That is right, more information is necessary. I am thinking why we have number of gameservers in fleet in E2E test is